test/XRP/issued-currency-client-integration.test.ts

```diff
@@ -240,6 +238,44 @@ describe('IssuedCurrencyClient Integration Tests', function (): void {
     assert.equal(transactionStatus2, TransactionStatus.Succeeded)
   })
 
+  // TODO: when SDK functionality is expanded, improve test specificity by creating trustlines/issued currencies first,
+  // which will also avoid the need for maintenance after a testnet reset.
+
+  // TODO: Implement an integration test for an account with balances/assets/obligations once functionality exists for first creating things.
+  it('getGatewayBalances - account not found', async function (): Promise<
+    void
+  > {
+    this.timeout(timeoutMs)
+    // GIVEN a valid address that doesn't actually exist on the ledger
+    const walletFactory = new WalletFactory(XrplNetwork.Test)
+    const address = (await walletFactory.generateRandomWallet())!.wallet.getAddress()
```

Comment:
Is this necessary when there's a wallet generated in `beforeEach`?

Reply:
so this one actually is because I want a valid address that is NOT a faucet wallet, (because a faucet wallet actually does exist on the ledger -- it has been funded).  Luckily, this makes no network calls and is fast.